lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgstr "MMU fordrar anv\xe1ndarens uppm\xe1rksamhet.."
+
+# MSG_MMU_POWER_FAILS c=15
+#: ultralcd.cpp:1517
+msgid "MMU power fails"
+msgstr "MMU str\xefmavbr"
+
+# MSG_STEALTH c=7
+#: messages.c:108
+msgid "Stealth"
+msgstr "Tyst"
+
+# MSG_AUTO_POWER c=10
+#: messages.c:105
+msgid "Auto power"
+msgstr "Auto kraft"
+
+# MSG_HIGH_POWER c=10
+#: messages.c:104
+msgid "High power"
+msgstr "H\xefg kraft"
+
+# MSG_MMU_CONNECTED c=18
+#: ultralcd.cpp:1997
+msgid "MMU2 connected"
+msgstr "MMU2 ansluten"
+
+# MSG_SELFTEST_MOTOR c=18
+#: messages.c:96
+msgid "Motor"
+msgstr "Motor"
+
+# MSG_MOVE_AXIS c=18
+#: ultralcd.cpp:5677
+msgid "Move axis"
+msgstr "Flytta axlar"
+
+# MSG_MOVE_X c=18
+#: ultralcd.cpp:4233
+msgid "Move X"
+msgstr "Flytta X"
+
+# MSG_MOVE_Y c=18
+#: ultralcd.cpp:4234
+msgid "Move Y"
+msgstr "Flytta Y"
+
+# MSG_MOVE_Z c=18
+#: ultralcd.cpp:4235
+msgid "Move Z"
+msgstr "Flytta Z"
+
+# MSG_NO_MOVE c=20
+#: Marlin_main.cpp:5852
+msgid "No move."
+msgstr "Ingen r\xefrelse."
+
+# MSG_NO_CARD c=18
+#: ultralcd.cpp:6697
+msgid "No SD card"
+msgstr "Inget SD-kort"
+
+# MSG_NA c=3
+#: messages.c:127
+msgid "N/A"
+msgstr "N/A"
+
+# MSG_NO c=4
+#: messages.c:71
+msgid "No"
+msgstr "Nej"
+
+# MSG_SELFTEST_NOTCONNECTED c=20
+#: ultralcd.cpp:8128
+msgid "Not connected"
+msgstr "Inte ansluten"
+
+# MSG_NEW_FIRMWARE_AVAILABLE c=20 r=2
+#: util.cpp:195
+msgid "New firmware version available:"
+msgstr "Ny firmware version tillg\xe1nglig:"
+
+# MSG_SELFTEST_FAN_NO c=19
+#: messages.c:92
+msgid "Not spinning"
+msgstr "Roterar inte"
+
+# MSG_WIZARD_V2_CAL c=20 r=8
+#: ultralcd.cpp:4838
+msgid "Now I will calibrate distance between tip of the nozzle and heatbed surface."
+msgstr "Nu ska jag kalibrera avstandet mellan munstyckets spets och v\xe1rmeb\xe1ddsytan."
+
+# MSG_WIZARD_WILL_PREHEAT c=20 r=4
+#: ultralcd.cpp:4967
+msgid "Now I will preheat nozzle for PLA."
+msgstr "Nu ska jag f\xefrv\xe1rma munstycket f\xefr PLA."
+
+# MSG_NOZZLE c=12
+#: messages.c:72
+msgid "Nozzle"
+msgstr "Munstycke"
+
+# MSG_DEFAULT_SETTINGS_LOADED c=20 r=6
+#: Marlin_main.cpp:1605
+msgid "Old settings found. Default PID, Esteps etc. will be set."
+msgstr "Gamla inst\xe1llningar hittades. Standard PID, Esteps etc. kommer att st\xe1llas in."
+
+# MSG_REMOVE_TEST_PRINT c=20 r=4
+#: ultralcd.cpp:4958
+msgid "Now remove the test print from steel sheet."
+msgstr "Ta nu bort testutskriften fran metallskivan."
+
+# MSG_NOZZLE_FAN c=10
+#: ultralcd.cpp:1446
+msgid "Nozzle FAN"
+msgstr "Muns fl\xe1kt"
+
+# MSG_PAUSE_PRINT c=18
+#: messages.c:74
+msgid "Pause print"
+msgstr "Pausa utskrift"
+
+# MSG_PID_RUNNING c=20
+#: ultralcd.cpp:1327
+msgid "PID cal."
+msgstr "PID kalibrering."
+
+# MSG_PID_FINISHED c=20
+#: ultralcd.cpp:1333
+msgid "PID cal. finished"
+msgstr "PID kalibrering klar"
+
+# MSG_PID_EXTRUDER c=17
+#: ultralcd.cpp:5799
+msgid "PID calibration"
+msgstr "PID kalibrering"
+
+# MSG_PINDA_PREHEAT c=20
+#: ultralcd.cpp:683
+msgid "PINDA Heating"
+msgstr "PINDA uppv\xe1rmning"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.